test/system/workarea/storefront/affirm_logged_in_checkout_system_test.rb

```diff
@@ -4,6 +4,7 @@ module Workarea
   module Storefront
     class AffirmLoggedInCheckoutSystemTest < Workarea::SystemTest
       include Storefront::SystemTest
+      include ActionView::Helpers::SanitizeHelper
```

Comment:
Why did you need this?

Reply:
ahh sorry this is a relic from my testing and I don't actually need it anymore (I was trying to see if `strip_tags` on the HTML i18n string would match with Capybara's `assert_text` assertion)